src/Tizen.NUI/src/public/Visuals/VisualObject/GradientVisual.cs

```diff
@@ -0,0 +1,227 @@
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public GradientVisual() : this(Interop.VisualObject.VisualObjectNew(), true)
+        {
+            NDalicPINVOKE.ThrowExceptionIfExists();
+        }
+
+        internal GradientVisual(global::System.IntPtr cPtr, bool cMemoryOwn) : this(cPtr, cMemoryOwn, cMemoryOwn)
+        {
+        }
+
+        internal GradientVisual(global::System.IntPtr cPtr, bool cMemoryOwn, bool cRegister) : base(cPtr, cMemoryOwn, cRegister)
+        {
+            Type = (int)Tizen.NUI.Visual.Type.Gradient;
+        }
+        #endregion
+
+        #region Visual Properties
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public Vector2 StartPosition
+        {
+            set
+            {
+                UpdateVisualProperty((int)Tizen.NUI.GradientVisualProperty.StartPosition, value);
+            }
+            get
+            {
+                Vector2 ret = new Vector2();
+                using (var propertyValue = GetCachedVisualProperty((int)Tizen.NUI.GradientVisualProperty.StartPosition))
+                {
+                    propertyValue?.Get(ret);
+                }
+                return ret;
+            }
+        }
+
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public Vector2 EndPosition
+        {
+            set
+            {
+                UpdateVisualProperty((int)Tizen.NUI.GradientVisualProperty.EndPosition, value);
+            }
+            get
+            {
+                Vector2 ret = new Vector2();
+                using (var propertyValue = GetCachedVisualProperty((int)Tizen.NUI.GradientVisualProperty.EndPosition))
+                {
+                    propertyValue?.Get(ret);
+                }
+                return ret;
+            }
+        }
+
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public Vector2 Center
+        {
+            set
+            {
+                UpdateVisualProperty((int)Tizen.NUI.GradientVisualProperty.Center, value);
+            }
+            get
+            {
+                Vector2 ret = new Vector2();
+                using (var propertyValue = GetCachedVisualProperty((int)Tizen.NUI.GradientVisualProperty.Center))
+                {
+                    propertyValue?.Get(ret);
+                }
+                return ret;
+            }
+        }
+
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public float Radius
+        {
+            set
+            {
+                UpdateVisualProperty((int)Tizen.NUI.GradientVisualProperty.Radius, value);
+            }
+            get
+            {
+                float ret = 0.0f;
+                using var propertyValue = GetCachedVisualProperty((int)Tizen.NUI.GradientVisualProperty.Radius);
+                propertyValue?.Get(out ret);
+                return ret;
+            }
+        }
+
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public List<float> StopOffset
```

Comment:
https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-type-members#names-of-properties

> ✔️ DO name collection properties with a plural phrase describing the items in the collection instead of using a singular phrase followed by "List" or "Collection".

For the given property naming rule of dotnet framework design guidelines, how about using plural phrase : `StopOffsets` and `StopColors`